src/ext/framework/help_commands.rs

```diff
@@ -21,18 +33,34 @@ pub fn with_embeds(ctx: &Context,
         let name = args.join(" ");
 
         for (group_name, group) in groups {
-            let mut found: Option<(&String, &Command)> = None;
+            let mut found: Option<(&String, &Arc<Command>)> = None;
```

Comment:
Could use `Option<(&String, &InternalCommand)>` here. Longer, but makes use of a predefined type